packages/web/src/scss/components/Card/_Card.scss

```diff
@@ -0,0 +1,152 @@
+// 1. Get ready for card link overlay.
+// 2. If there is a expanded CardMedia in a boxed vertical card, replace card padding with empty grid columns/rows.
+// 3. Allow shrinking in grid layouts.
+// 4. Make sure links and buttons are clickable.
+// 5. Make text content selectable when there is a stretched CardLink.
+
+@use '@tokens' as tokens;
+@use 'sass:map';
+@use '../../tools/breakpoint';
+@use '../../tools/typography';
+@use 'theme';
+
+.Card {
+    --#{tokens.$css-variable-prefix}card-padding: #{theme.$padding-mobile};
+
+    @include typography.generate(theme.$typography);
+
+    position: relative; // 1.
+    display: grid;
+    min-width: 0; // 3.
+    color: theme.$color;
+
+    @include breakpoint.up(map.get(theme.$breakpoints, tablet)) {
```

Comment:
or you can pass the `tokens.$breakpoint-tablet` token directly here from theme and you don't need the sass:map here 🤷